.github/workflows/pr-screenshots.yml

```
@@ -0,0 +1,123 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+      
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+      
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)